secure/storage/src/tests/vault.rs

```diff
@@ -210,61 +308,197 @@ fn test_vault_crypto_policies() {
 
     // Verify exporter policy
     let exporter_token = storage.create_token(vec![&EXPORTER]).unwrap();
-    let mut exporter_store =
-        VaultStorage::new(dev::test_host(), exporter_token, None, None, None, true);
+    let mut exporter_store = VaultStorage::new(
+        dev::test_host(),
+        exporter_token.clone(),
+        None,
+        None,
+        None,
+        true,
+    );
     exporter_store.export_private_key(CRYPTO_KEY).unwrap();
     exporter_store.get_public_key(CRYPTO_KEY).unwrap_err();
     exporter_store.rotate_key(CRYPTO_KEY).unwrap_err();
     exporter_store.sign(CRYPTO_KEY, &message).unwrap_err();
+    // Verify a exporter with another namespace has no permission for the operations
+    let mut exporter_store_with_namespace = VaultStorage::new(
+        dev::test_host(),
+        exporter_token,
+        Some(VAULT_NAMESPACE_1.into()),
+        None,
+        None,
+        true,
+    );
+    exporter_store_with_namespace
```

Comment:
It's probably only worth checking that "export_private_key" didn't work. The rest won't work even with the correct namespace (due to permissions -- see the calls above).